permissions/plugin-synopsys-sigma.yml

```diff
@@ -0,0 +1,7 @@
+---
+name: "synopsys-sigma"
+github: "jenkinsci/synopsys-sigma-plugin"
+paths:
+- "org/jenkins-ci/plugins/synopsys-sigma"
```

Comment:
this is not right your group id in build.gradle is `io.jenkins.plugins` but your actual package path you've used is `com.synopsys.integration.jenkins.sigma`

Reply:
Sorry, I wasn't sure what was meant by the group.id, I see now. I've changed this to be `"io/jenkins/plugins/synopsys-sigma"` now. Is that all that is needed?